spec/models/school_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe School, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
You don't have tests here, so you can remove this file ✂️ 

EDIT: 
I can't add comment to `spec/support/features_helper.rb` file because it's empty. Remove it too :)